### ISSUE TITLE
[FEATURE] Is it possible to window rewrite workspace icons?

### ISSUE THREAD

**Issue by j5onrf** [Jas-SinghFSU/HyprPanel#296]
**Describe the solution you'd like**
When you visit a website the icon changes dynamic.

```
Waybar example: using "title"

"window-rewrite": {
		"title<.*youtube.*>": "",
                "class<firefox>": ""
}
```

**Additional context**
```
{
  "1": { "icon": "󰄛", "title<.*youtube.*>": ""},
}

{"1":"title<.*youtube.*>": "","2":"󰝰","3":"","4":"","5":""}
```
```
Window Titles
[
  ["youtube", "󰄛", "Youtube"],
]
```

**Comment by Jas-SinghFSU** [comment]
You want to show the workspace icon based on the last active window in the workspace (defined as a regex)? Or are you talking about the window title? Or both lol.

I can see this being useful, it's just that we should probably split this into two feature requests if that's the case.

**Comment by Jas-SinghFSU** [comment]
implemented by #368
